lib/axios.ts

```diff
@@ -3,15 +3,16 @@ const logger = require('./log');
 import axiosRetry from 'axios-retry';
 
 const axiosInstance = axios.create({
-  timeout: 5000,
+  timeout: 1000,
```

Comment:
@aarlaud, since we use `axiosInstance` everywhere for API calls, is 1000 not low?